app/models/dtos.py

```diff
@@ -33,7 +33,7 @@ class Message(BaseModel):
         sent_at: datetime = Field(
             alias="sentAt", default_factory=datetime.utcnow
         )
-        content: Content
+        content: list[Content]
```

Comment:
```suggestion
        contents: list[Content]
```
because it's a list I would suggest to use plural

Reply:
This is how Artemis uses it, so it has to be like this here as well.